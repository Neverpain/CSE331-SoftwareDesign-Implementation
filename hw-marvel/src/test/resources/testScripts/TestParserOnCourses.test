# Tests the makeGraph method on the courses file to see that it is
# built correctly, prints out nodes and children of nodes

LoadGraph g courses.csv

ListNodes g
ListChildren g cse331
ListChildren g math124
ListChildren g eng101