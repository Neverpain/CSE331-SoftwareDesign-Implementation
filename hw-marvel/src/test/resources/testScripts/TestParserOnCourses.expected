# Tests the makeGraph method on the courses file to see that it is
# built correctly, prints out nodes and children of nodes

loaded graph g

g contains: cse142 cse143 cse331 cse391 eng101 eng152 eng191 math124 math125 math126
the children of cse331 in g are: cse142(cse) cse143(cse) cse391(cse)
the children of math124 in g are: math125(math) math126(math)
the children of eng101 in g are: eng152(english) eng191(english)